### PR TITLE
feat: DAH-2731 add potrero question

### PR DESCRIPTION
**Pull request by tallulahkay**
## Description

Adds application question for Potrero Right to Return listing.

## Jira ticket

https://sfgovdt.jira.com/browse/DAH-2731

### Version Control

- [x] branch name begins with `angular` if it contains updates to Angular code
- [x] branch name contains the Jira ticket number
- [x] PR name follows `type: TICKET-NUMBER Description` format, e.g. `feat: DAH-123 New Feature`

### Code quality

- [ ] [the set of changes is small](https://google.github.io/eng-practices/review/developer/small-cls.html#what-is-small)
- [x] all automated code checks pass (linting, tests, coverage, etc.)
- [x] code irrelevant to the ticket is not modified e.g. changing indentation due to automated formatting
- [x] if the code changes the UI, it matches the UI design exactly

### Review instructions

- [x] instructions specify which environment(s) it applies to
- [x] instructions work for PA testers
- [x] instructions have already been performed at least once

### Request review

- [x] PR has `needs review` label
- [x] Use `Housing Eng` group to automatically assign reviewers, and/or assign specific engineers
- [ ] If time sensitive, notify engineers in Slack

### CODE REVIEW

File: app/assets/json/translations/locale-en.json [hunk 1]
```diff
@@ -1275,6 +1275,13 @@
                 "proof_desc": "You'll need one of the following documents showing your address at Hunters View public housing.",
                 "title": "Right to Return - Hunters View (West Point)"
             },
+            "rtr_potrero": {
+                "address": "Potrero Terrace and Annex Address",
+                "address_desc": "Your current or former address at Potrero Terrace and Annex public housing",
+                "desc": "For households in which at least one person is a former or current resident of Potrero Terrace and Annex public housing. Please note, these units are not subsidized, the rent will not change if your income changes. ",
+                "proof_desc": "You'll need one of the following documents showing your address at Potrero Terrace and Annex public housing.",
+                "title": "Right to Return - Potrero Terrace and Annex"
```
Comment (jimlin-sfgov) [inline app/assets/json/translations/locale-en.json:1283]: question: The figma designs use ""Right to Return - Potrero Terrace and Annex public housing", but that doesn't follow the pattern of `rtr_huntersview`. Can you confirm which title is correct?

Comment (tallulahkay) [reply app/assets/json/translations/locale-en.json:1283]: Bridget confirmed, but I just have to make an update to one spot

Comment (tallulahkay) [reply app/assets/json/translations/locale-en.json]: @jimlin-sfgov I updated this - should match the design now

**Review by jimlin-sfgov — APPROVED** [review]
LGTM

**Pull request merged**